packages/compass-generative-ai/src/atlas-ai-service.ts

```diff
@@ -437,6 +442,30 @@ export class AtlasAiService {
     );
   }
 
+  // optIn function w post request to set the user preference to true
+  async optIntoGenAIFeaturesAtlas() {
+    const res = await this.atlasService.authenticatedFetch(
+      this.atlasService.cloudEndpoint(
+        '/settings/optInDataExplorerGenAIFeatures'
+      ),
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          isEnabled: true,
```

Comment:
On the endpoint this should be `value`:
https://github.com/10gen/mms/pull/112492/files#diff-46c26bd876b988becd6fb63bd05028cc6747d74a3d8b4dd6d0728b043e09bd33R820 
Here's a related example: https://github.com/10gen/mms/blob/9089273fb2c9121547b4f54f799d40f06dbb47c2/client/packages/common/services/api/settingsApi.ts#L87

Reply:
done!